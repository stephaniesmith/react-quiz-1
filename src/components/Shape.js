import React from 'react';
import PropTypes from 'prop-types';

const Shape = ({ text, color, backgroundColor }) => {
  return (
    <div style={{ width: '100px', height: '100px', color, backgroundColor }}>
      <h2>{text}</h2>
    </div>
  );
};

Shape.propTypes = {
  text: PropTypes.string,
  color: PropTypes.string,
  backgroundColor: PropTypes.string
};

export default Shape;
