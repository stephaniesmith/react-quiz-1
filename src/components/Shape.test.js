import React from 'react';
import { shallow } from 'enzyme';
import Shape from './Shape';

describe('Shape', () => {
  it('renders Shape', () => {
    const wrapper = shallow(<Shape />);
    expect(wrapper).toMatchSnapshot();
  });
});
