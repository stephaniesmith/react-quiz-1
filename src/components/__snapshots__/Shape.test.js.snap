// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape renders Shape 1`] = `
<div
  style={
    Object {
      "backgroundColor": undefined,
      "color": undefined,
      "height": "100px",
      "width": "100px",
    }
  }
>
  <h2 />
</div>
`;
